packages/fluent-editor/src/utils/is.ts

```
@@ -1,5 +1,7 @@
-export function isObject(value): value is object {
+export function isObject(value: unknown): value is object {
   const type = typeof value
   return value != null && (type === 'object' || type === 'function')
 }
 export const isUndefined = (val: unknown): val is undefined => val === undefined
+export const isFunction = (val: unknown): val is Function => typeof val === 'function'
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)